tests/mac-eve-multiple-disabled/suricata.yaml

```diff
@@ -0,0 +1,11 @@
+%YAML 1.1
+---
+
+outputs:
+  - eve-log:
+      enabled: yes
+      filetype: regular #regular|syslog|unix_dgram|unix_stream|redis
```

Comment:
maybe remove the comment part as it's not necessary :)

Reply:
Sure, the cleaner the better!